### DISCUSSION TITLE
How does SK compare to its competitors (e.g. LangChain, LlamaIndex)?

### DISCUSSION THREAD

**Discussion by suerta-git** [microsoft/semantic-kernel#272] [General]
What are the advantages of SK compared to others?

At least SK is the best and only option in .NET ecosystem lol.

**Comment by timlaverty** [comment]
Hi suerta- we haven't done a detailed comparison.  This space is moving very quickly and our objective has been to move as fast as possible reacting to feedback and feature asks from devs working in any language.

**Comment by tonyqus** [comment]
https://github.com/tonyqus/semantic-kernel-vs-langchain
